plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -936,6 +938,24 @@ def read(self, file_name):
         if nodes is None:
             nodes = []
 
+        if self._is_ucp:
+            # We might be on a different printer than the one this project was made on.
+            # The offset to the printers' center isn't saved; instead, try to just fit everything on the buildplate.
+            full_extents = None  # Don't initialize to 'Null'!
```

Comment:
I don't quite get this comment, python has no `null` value, and it's closest equivalent is `None`. and the value is initialized to `None`
```suggestion
            full_extents = None
```

Reply:
I'm fine with removing it, but for posterities sake:

The code didn't work when initializing with an 'empty' AABB, which _should_ have been set up so that this _does_ work, at least in my opinion. _Inside_ the AABB the vectors where set to a constant called 'Null' (yes, not 'None'), which is 0,0,0 (should have been 'infinite in the other direction' for min/max).

Anyway, I get why my comment was insufficient to communicate that 😅